### PR TITLE
[Shop][Behat] Payment step

### PR DESCRIPTION
**Pull request by mpysiak**
| Q               | A
|-----------------|-----
| Branch?         | bootstrap-shop
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | n/a
| License         | MIT

### CODE REVIEW

**Comment by mpysiak** [comment]
I did not enable `Skipping payment method step` scenarios (and some other single scenarios) because they require checkout complete to be ready

File: src/Sylius/Behat/Context/Ui/Shop/Checkout/CheckoutPaymentContext.php [hunk 1]
```diff
@@ -189,4 +189,34 @@ public function iShouldNotSeeAndPaymentMethods(string ...$paymentMethodsNames):
             );
         }
     }
+
+    /**
+     * @Given I do not select any payment method
```
Comment (Rafikooo) [inline src/Sylius/Behat/Context/Ui/Shop/Checkout/CheckoutPaymentContext.php]:
```suggestion
     * @When I do not select any payment method
```
And it should be moved above `@Then` methods.

**Review by Rafikooo — APPROVED** [review]

File: features/shop/checkout/paying_for_order/preventing_payment_step_completion_without_a_selected_method.feature [hunk 2]
```diff
@@ -10,7 +10,7 @@ Feature: Preventing payment step completion without a selected method
         And the store has a product "PHP T-Shirt" priced at "$19.99"
         And I am a logged in customer
 
-    @todo @ui @todo
+    @no-api @ui @javascript
```
Comment (GSadee) [inline features/shop/checkout/paying_for_order/preventing_payment_step_completion_without_a_selected_method.feature]: I guess that this scenario and below ones could make sense in API context, so add here `@todo-api` tag 🤔

**Review by GSadee — APPROVED** [review]

File: features/shop/checkout/prices_get_updated_throughout_checkout.feature [hunk 3]
```diff
@@ -51,7 +51,7 @@ Feature: Prices get updated when exchange rate changes during the whole checkout
         And I decide to change order shipping method
         Then the subtotal of "The Pug Mug" item should be "£30.00"
 
-    @no-api @todo-ui
+    @no-api @todo-ui #requires implementation of checkout complete
```
Comment (Wojdylak) [inline features/shop/checkout/prices_get_updated_throughout_checkout.feature:54]: Could you add information about these tests in the checkout complete task?

**Review by Wojdylak — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Pull request merged**